charts/seed-bootstrap/templates/elasticsearch/es-statefulset.yaml

```diff
@@ -0,0 +1,64 @@
+# Elasticsearch deployment itself
+apiVersion: {{ include "statefulsetversion" . }}
+kind: StatefulSet
+metadata:
+  name: elasticsearch-logging
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: elasticsearch-logging
+    role: logging
+spec:
+  serviceName: elasticsearch-logging
+  selector:
+    matchLabels:
+      app: elasticsearch-logging
+      role: logging
+  template:
+    metadata:
+      labels:
+        app: elasticsearch-logging
+        role: logging
+    spec:
+      containers:
+      - name: elasticsearch-logging
+        image: {{ index .Values.images "elasticsearch-oss" }}
+        imagePullPolicy: IfNotPresent
+        resources:
+          # need more cpu upon initialization, therefore burstable class
+          limits:
+            cpu: 1000m
+            memory: 3Gi
+          requests:
+            cpu: 100m
+            memory: 2Gi
+        ports:
+        - containerPort: {{ .Values.elasticsearchPorts.db }}
+          name: db
+          protocol: TCP
+        - containerPort: {{ .Values.elasticsearchPorts.transport }}
+          name: transport
+          protocol: TCP
+        volumeMounts:
+        - name: elasticsearch-logging
+          mountPath: /data
+      volumes:
+      - name: elasticsearch-logging
+        emptyDir: {}
+      # Elasticsearch requires vm.max_map_count to be at least 262144.
+      # If your OS already sets up this number to a higher value, feel free
+      # to remove this init container.
+      initContainers:
+      - image: {{ index .Values.images "alpine" }}
+        command: ["/sbin/sysctl", "-w", "vm.max_map_count=262144"]
```

Comment:
Can we set this value only if the the current value of `vm.max_map_count` is less than `262144`?

Reply:
Yes, fixed.